packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -95,7 +93,7 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
     return (
       <div
         { ...divProps }
-        className={ css('ms-Persona', styles.root, className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence as PersonaPresenceEnum], this.props.showSecondaryText ? styles.showSecondaryText : '') }
+        className={ css('ms-Persona-coin') }
```

Comment:
By removing these styles from PersonaCoin, FacePile doesn't render Persona sizes correctly. 
![facepile](https://user-images.githubusercontent.com/15041132/30763937-82b10bfe-9f9d-11e7-8c0a-2c7510e5eddf.PNG)

I believe the issue is that height of image is based on nested styles, e.g. 
```scss
.root.rootIsExtraExtraSmall {
  height: $ms-Persona-sizeXs;
  min-width: $ms-Persona-sizeXs;

  .imageArea,
  .image {
    flex: 0 0 $ms-Persona-sizeXxs;
    height: $ms-Persona-sizeXxs;
    width: $ms-Persona-sizeXxs;
  }
}

Without the `root` style on PersonaCoin, height defaults to `$ms-Persona-sizeMd`

Also, `ms-Persona-coin` is not defined. Did you mean to add styling for it?

Reply:
Great catch Erich. I changed .rootIsExtraExtraSmall to not be tied to the root for imageArea, image, initials etc. This way I can also put the class on the coin for instances that it's used on it's own.